tests/Type/Doctrine/Query/QueryResultTypeWalkerTest.php

```diff
@@ -208,1126 +229,1247 @@ public function test(Type $expectedType, string $dql, ?string $expectedException
 	}
 
 	/**
-	 * @return array<array-key,array{Type,string,2?:string}>
+	 * @return iterable<string,array{Type,string,2?:string}>
 	 */
-	public function getTestData(): array
+	public function getTestData(): iterable
```

Comment:
Changed this data provider to `yield` test cases so that I could return some cases conditionally.

Best reviewed with `?w=1` :D